backend/app/api/quiz.py

```
@@ -0,0 +1,71 @@
+from typing import List
+from fastapi import APIRouter, Depends, HTTPException
+from sqlmodel import Session
+
+from app.crud import quiz as crud_quiz
+from app.schemas import quiz as quiz_schemas
+from app.models import quiz as quiz_models
```

Ruff (0.12.2)

`app.models.quiz` imported but unused

Remove unused import: `app.models.quiz`

(F401)